hoomd/md/methods.py

```diff
@@ -454,14 +454,38 @@ def thermalize_thermostat_and_barostat_dof(self, seed):
             seed, self._simulation.timestep)
 
 
-class nph(NPT):
+class NPH(NPT):
     R""" NPH Integration via MTK barostat-thermostat..
```

Comment:
```suggestion
    r"""NPH Integration via MTK barostat-thermostat.
```
PEP8 dictated style.